components/faqs/nesting.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import { withTheme, ThemeProvider } from 'styled-components'
+
+import SectionLayout from '../SectionLayout'
+import CodeBlock from '../CodeBlock'
+import LiveEdit from '../LiveEdit'
+
+const scope = { withTheme, ThemeProvider }
+const sample = (`
+const ColorChanger = styled.section\`
+  background: papayawhip;
+
+  > h2 {
+    color: palevioletred;
+  }
+
+  @media(min-width: 768px) {
+    background: mediumseagreen;
+
+    > h2 {
+      color: papayawhip;
+    }
+  }
+\`;
+render(
+  <ColorChanger href="#">
+    <h2>Hello world!</h2>
+  </ColorChanger>
+)
+`).trim()
+
+const Nesting = () => (
+  <SectionLayout title="Nesting">
+    <p>Nesting is a feature ported from Sass. Used sparingly it's a great way to lighten your code by reducing the need to create explicit classes for every element.</p>
```

Comment:
I've always liked this example for nesting:

```js
const EqualDivider = styled.div`
  display: flex;
  padding: 1rem;
  ${ props => props.vertical && 'flex-direction: vertical;'  }

  > * {
    flex-grow: 1;

    &:not(:first-child) {
      ${ props => props.vertical ? 'margin-top' : 'margin-left' }: 1rem;
    }
  }
`
```

Adapted from this slide from [this talk](https://www.youtube.com/watch?v=XR6eM_5pAb0):

![image](https://cloud.githubusercontent.com/assets/23264/26748612/8f9483b2-4842-11e7-8724-103d03247125.png)